robottelo/vm.py

```diff
@@ -375,10 +375,11 @@ def configure_rhai_client(self, activation_key, org, rhel_distro):
         service.
 
         :param activation_key: Activation key to be used to register the
-        system to satellite
+            system to satellite
         :param org: The org to which the system is required to be registered
         :param rhel_distro: rhel distribution for
         :return: None
+
```

Comment:
No need to add this blank line.

For more info check: https://www.python.org/dev/peps/pep-0008/#documentation-strings

PS: I know that we have been doing this for a long time, but some time ago I read again that section and figured out that we ensured a non PEP8 standard.